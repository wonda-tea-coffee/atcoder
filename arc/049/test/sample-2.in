Niwawo_Kakemeguru_Chokudai
11 17 18 26
