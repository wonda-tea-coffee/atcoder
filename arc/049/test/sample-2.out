Niwawo_Kake"meguru"_"Chokudai"
